numpy/lib/twodim_base.py

```diff
@@ -32,6 +31,75 @@ def _min_int(low, high):
     return int64
 
 
+def flip(m, axis):
+    """
+    Reverse the order of elements in an array along the given axis.
+
+    The shape of the array is preserved, but the elements are reordered.
+
```

Comment:
```rot90``` uses ```flip``` so it is based on the other PR. After squashing the commits on the ```flip``` PR, we will rebase the ```rot90``` branch on the ```flip``` branch.